src/Gui/DockWindowManager.cpp

```diff
@@ -424,7 +424,8 @@ void DockWindowManager::activate(QWidget* widget)
         dw->toggleViewAction()->activate(QAction::Trigger);
     }
 
-    dw->raise();
+    if (setFocus)
+        dw->raise();

```

Comment:
This file is not too good at following that, but all ifs should have braces:
```suggestion
    if (setFocus) {
        dw->raise();
    }
```